diff --git utils/bax2bam/src/CMakeLists.txt utils/bax2bam/src/CMakeLists.txt
index ee67c76..2c6b8f0 100644
--- utils/bax2bam/src/CMakeLists.txt
+++ utils/bax2bam/src/CMakeLists.txt
@@ -49,4 +49,5 @@ target_link_libraries(bax2bam
     ${PacBioBAM_LIBRARIES}
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
+    -lrt
 )
diff --git utils/bax2bam/tests/CMakeLists.txt utils/bax2bam/tests/CMakeLists.txt
index 1621169..64e57e3 100644
--- utils/bax2bam/tests/CMakeLists.txt
+++ utils/bax2bam/tests/CMakeLists.txt
@@ -46,6 +46,7 @@ target_link_libraries(test_bax2bam
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT} # quirky pthreads
+    -lrt
 )
 # add_test(test_bax2bam test_bax2bam)
 add_test(
diff --git utils/bam2bax/src/CMakeLists.txt utils/bam2bax/src/CMakeLists.txt
index 8838b23..9ee0671 100644
--- utils/bam2bax/src/CMakeLists.txt
+++ utils/bam2bax/src/CMakeLists.txt
@@ -54,6 +54,7 @@ target_link_libraries(bam2bax
     ${PacBioBAM_LIBRARIES}
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
+    -lrt
 )
 
 add_executable(bam2plx ${BAM2PLX_SOURCES} )
@@ -69,4 +70,5 @@ target_link_libraries(bam2plx
     ${PacBioBAM_LIBRARIES}
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
+    -lrt
 )
diff --git utils/bam2bax/tests/CMakeLists.txt utils/bam2bax/tests/CMakeLists.txt
index b9dfff7..dbc54e6 100644
--- utils/bam2bax/tests/CMakeLists.txt
+++ utils/bam2bax/tests/CMakeLists.txt
@@ -46,6 +46,7 @@ target_link_libraries(test_bam2bax
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT} # quirky pthreads
+    -lrt
 )
 
 add_test(
