diff --git utils/bax2bam/src/CMakeLists.txt utils/bax2bam/src/CMakeLists.txt
index ee67c76..2c6b8f0 100644
--- utils/bax2bam/src/CMakeLists.txt
+++ utils/bax2bam/src/CMakeLists.txt
@@ -49,4 +49,5 @@ target_link_libraries(bax2bam
     ${PacBioBAM_LIBRARIES}
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
+    -lrt
 )
diff --git utils/bax2bam/tests/CMakeLists.txt utils/bax2bam/tests/CMakeLists.txt
index 1621169..64e57e3 100644
--- utils/bax2bam/tests/CMakeLists.txt
+++ utils/bax2bam/tests/CMakeLists.txt
@@ -46,6 +46,7 @@ target_link_libraries(test_bax2bam
     ${HTSLIB_LIBRARIES}
     ${ZLIB_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT} # quirky pthreads
+    -lrt
 )
 # add_test(test_bax2bam test_bax2bam)
 add_test(
